modules/osdk-how-csv-gen-works.adoc

```diff
@@ -5,30 +5,26 @@
 [id="osdk-how-csv-gen-works_{context}"]
 = How CSV generation works
 
-The `deploy/` directory of an Operator project is the standard location for all manifests required to deploy an Operator. The Operator SDK can use data from manifests in `deploy/` to write a cluster service version (CSV).
+The `manifests/` directory of an Operator project is the standard location for all manifests required to deploy an Operator on OLM. The CSV generator, called by the `generate bundle` subcommand, requires certain input manifests to construct a CSV manifest; all inputs are read when the command is invoked, along with a CSV base, to idempotently regenerate a CSV.
 
-The following command:
+The following resource kinds are typically included in a CSV:
 
-[source,terminal]
-----
-$ operator-sdk generate csv --csv-version <version>
-----
+* `Role`: Define Operator permissions within a namespace.
```

Comment:
This might make a great definition list.  They're also fragments, so you don't need the end period.